distribution/osbuild-worker@.service

```diff
@@ -7,7 +7,7 @@ After=multi-user.target osbuild-composer.socket
 Type=simple
 PrivateTmp=true
 ExecStart=/usr/libexec/osbuild-composer/osbuild-worker
-CacheDirectory=osbuild-composer
+CacheDirectory=osbuild-worker
```

Comment:
I'd prefer not to have it here if we don't use it. It seems to me that it is just confusing and the effort to remove it and reintroduce it is minimal.

Reply:
Yeah, I don't mind. I'm happy to merge a PR that drops it. I just did the minimal change for now to move it out of the way (sharing it caused problems with the wrong owner/group).

Reply:
Correction, I'll drop this. I have to rebase on top of your RCM PR anyway.